src/app/globals.css

```
@@ -1,3 +1,19 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.gradient {
+    @apply bg-clip-text bg-gradient-to-r from-[#00bfa6] from-35% via-[#00bfa6bd] via-65% to-[#00bfa6a5] to-100% text-transparent;
+}
+
+.bg-page {
+    @apply dark:bg-black bg-white;
+}
+
+.text-page {
+    @apply dark:text-white text-black;
```

Same with texts! Keep the contrast moderate, our eyes will thanks us! :eyes: :frog:

```suggestion
@apply dark:text-[#FCFCFC] text-[#0D0D0D];
```

Just while we get a defined color palettte